CHANGES.rst

```diff
@@ -81,6 +81,9 @@ alma
 
 gaia
 ^^^^
+
+- Update documentation to show how to control the number of rows returned by
```

Comment:
Please remove the changelog

a rule of thumb is that there is  no need for a changelog for changes to the docs, or internals that has non direct effect on user facing behaviour.